src/WebJobs.Script/Rpc/ILanguageWorkerChannel.cs

```diff
@@ -19,6 +19,8 @@ public interface ILanguageWorkerChannel : IDisposable
 
         WorkerConfig Config { get; }
 
+        void SetupFunctionInvocationBuffers(IEnumerable<FunctionMetadata> functions);
```

Comment:
This seems pretty specific for an interface... can it be `Initialize()` or something?

Also -- it looks very similar to `RegisterFunctions()` -- can this be done there already?

Reply:
> This seems pretty specific for an interface... can it be Initialize() or something?

This is needs to be an explicit action by the function dispatcher specifically for a language worker channel that is established during placeholder mode. In placeholder mode, functions are not loaded by the language worker. 

> Also -- it looks very similar to RegisterFunctions() -- can this be done there already?

RegisterFunctions() is sending function load messages to language worker. Will rename to reflect the action.